test/test_assets/Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
```

Comment:
By adding a Gemfile, we introduce a hard dependency on ruby, rubygems, and bundler in order to run tests. We should document this in README.md or EXTEND.md.